extension/data/modules/macros.js

```diff
@@ -163,7 +178,7 @@ function modmacros () {
                     const macroButtonHtml = `<select class="tb-macro-select tb-action-button" data-subreddit="${info.subreddit}"><option value=${MACROS}>macros</option></select>`;
                     $body.find('.ThreadViewerReplyForm__replyOptions').after(`<div class="tb-usertext-buttons tb-macro-newmm">${macroButtonHtml}</div>`);
 
-                    populateSelect('.tb-macro-select', info.subreddit, config);
+                    populateSelect('.tb-macro-select', info.subreddit, config, 'comment');
```

Comment:
Unless I'm misunderstanding something, this line requires macros to have the "comment" type to work in new modmail, which seems wrong. Changing it to "modmail" fixes it in the case I was testing, at least. Would like @SpyTec to check my logic here though.

```suggestion
                    populateSelect('.tb-macro-select', info.subreddit, config, 'modmail');
```

Reply:
Good catch, double checked and you're right